mne/io/eeglab/eeglab.py

```diff
@@ -244,7 +257,7 @@ class RawEEGLAB(_BaseRaw):
     """
     @verbose
     def __init__(self, input_fname, montage, preload=False, eog=(),
-                 verbose=None):
+                 event_id=dict(), verbose=None):
```

Comment:
so you do nothing by default? If the user doesn't give a mapping, he can't read the file? That's a bit weird ... and that's also the reason your test fails